_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-instagram.php

```diff
@@ -99,7 +99,7 @@ public function get_instagram_connect_url() {
 			return $response;
 		}
 		$body        = json_decode( wp_remote_retrieve_body( $response ) );
-		$connect_url = $body->services->instagram->connect_URL;
+		$connect_url = $body->services->{'instagram-basic-display'}->connect_URL;
```

Comment:
Is this an unrelated change?

Reply:
This is unrelated but also likely cause for your issue in the #15470 (see https://github.com/Automattic/jetpack/pull/15470#issuecomment-615308894).

I'm fine with having this here, as it would otherwise be impossible to test it.

I'll also make sure to rebase #15470 once this is in.